rclcpp_lifecycle/test/benchmark/benchmark_lifecycle_node.cpp

```diff
@@ -131,6 +131,19 @@ BENCHMARK_F(BenchmarkLifecycleNode, get_available_transitions)(benchmark::State
   }
 }
 
+BENCHMARK_F(BenchmarkLifecycleNode, get_transition_graph)(benchmark::State & state) {
+  for (auto _ : state) {
+    constexpr size_t expected_transitions = 25u;
+    const auto & transitions = node->get_transition_graph();
+    if (transitions.size() != expected_transitions) {
+      const std::string msg = std::to_string(transitions.size());
```

Comment:
I know you're just mirroring the other benchmarks (which I think I wrote), but I think this error message could be a little more detailed. Maybe something like:
```suggestion
      const std::string msg = std::string("Expected number of transitions did not match actual: ") + std::to_string(transitions.size());
```